storm-core/src/jvm/org/apache/storm/utils/TopologySpoutLag.java

```diff
@@ -68,14 +77,38 @@
         commands.add((String) jsonConf.get(GROUPID_CONFIG));
         commands.add("-b");
         commands.add((String) jsonConf.get(BOOTSTRAP_CONFIG));
-        String securityProtocol = (String) jsonConf.get(CONFIG_KEY_PREFIX + "security.protocol");
+        String securityProtocol = (String) jsonConf.get(SECURITY_PROTOCOL_CONFIG);
         if (securityProtocol != null && !securityProtocol.isEmpty()) {
             commands.add("-s");
             commands.add(securityProtocol);
         }
         return commands;
     }
 
+    private static File getExtraPropertiesFile(Map<String, Object> jsonConf) {
```

Comment:
nit: maybe using `create` or `build` or so instead of `get` would be clear to represent that new (temporary) file is generated.